open_access/synapse_to_cds.md

```diff
@@ -0,0 +1,69 @@
+
+---
+order: 993
+---
+
+# Guide: Transferring Uncontrolled Data from the HTAN Data Portal to the Cancer Genomics Cloud (CGC)
+
+## Screen Recording Walkthrough
+**[HTAN Data Jamboree Tutorial: Getting Uncontrolled Data from the HTAN Portal to SB-CGC](#)**
```

Comment:
missing a link in case the embed does not work

Reply:
https://us02web.zoom.us/clips/share/JNSyNG3rPRoWVYjWrYNiV-Z9Tv-TYIKdb54YLB6pk6CRhhnEQ0g14a4-2hItZXGOYXq3CzcSVjgNTL7HMhHjUm72.cd78QzhYRY4j02Mw